src/Symfony/Component/DependencyInjection/Tests/ContainerTest.php

```diff
@@ -178,7 +168,6 @@ public function testGetThrowServiceNotFoundException()
     {
         $sc = new ProjectServiceContainer();
         $sc->set('foo', $foo = new \stdClass());
-        $sc->set('bar', $foo = new \stdClass());
```

Comment:
This one was only added to match up with the expected exception message (including "bar" as alternative), which i fixed in `Container`.